pymatgen/analysis/quasirrho.py

```diff
@@ -0,0 +1,206 @@
+# coding: utf-8
+# Copyright (c) Pymatgen Development Team.
+# Distributed under the terms of the MIT License.
+
+
+"""
+A module to calculate free energies using the Quasi-Rigid Rotor Harmonic
+ Oscillator approximation. Modified from a script by Steven Wheeler.
+See: Grimme, S. Chem. Eur. J. 2012, 18, 9955
+"""
+
+__author__ = "Alex Epstein"
+__copyright__ = "Copyright 2020, The Materials Project"
+__version__ = "0.1"
+__maintainer__ = "Alex Epstein"
+__email__ = "aepstein@lbl.gov"
+__date__ = "January 5, 2021"
+__credits__ = "Steven Wheeler, Trevor Seguin"
+
+from typing import Union
+
+import numpy as np
+from pymatgen.io.gaussian import GaussianOutput
+from pymatgen.io.qchem.outputs import QCOutput
+
+# Define useful constants
+kb = 1.380662E-23  # J/K
```

Comment:
Are these already defined in pymatgen.util.units?

Reply:
Looks like yes! I was not aware of pymatgen units -- will fix and commit the changes

Reply:
A lot of the units I'm using are slightly different and don't lend themselves well to the subclasses in pymatgen.core.units. For example, converting amu*angs^2 to kg*m^2 for moments of inertia. Would you recommend still using Units subclasses when I can?

Reply:
I would recommend supplementing the existing units if you don't have the ones you want.

To be specific, I'm talking about the units *constants* in pymatgen.core.units specifically, rather than suggesting you use the classes in that file.

The reason for this recommendation is simply that if we expand our unit functionality later (we've explored a few options here), it's a lot easier for us to spot where units are being used in pymatgen if unit constants are imported from the same place.